jetstream/core/metrics/prometheus.py

```diff
@@ -37,21 +36,46 @@ def __new__(cls):
       documentation="Size of prefill queue",
       labelnames=["id"],
   )
+
   _transfer_backlog = Gauge(
       name="jetstream_transfer_backlog_size",
       documentation="Size of transfer queue",
       labelnames=["id", "idx"],
   )
+
   _generate_backlog = Gauge(
       name="jetstream_generate_backlog_size",
       documentation="Size of generate queue",
       labelnames=["id", "idx"],
   )
+
+  _queue_duration = Histogram(
+      name="jetstream_queue_duration",
+      documentation="The total time each request spends enqueued in seconds",
+      labelnames=["id"],
+      buckets=[
+          0.01,
+          0.02,
+          0.05,
+          0.1,
+          0.2,
+          0.5,
+          1.0,
+          2.0,
+          5.0,
+          10.0,
+          20.0,
+          50.0,
+          100.0,
+      ],
+  )
```

Comment:
Why we use this percentage buckets?

Reply:
There isn't a standard for queue duration buckets, the 1-2-5 pattern for buckets is frequent when this is the case